examples/rotator_test_light.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
```

Comment:
This needs a description of what the test does and how it's meant to be launched. It's also not a notebook... does this really belong here?

Reply:
It used to be a notebook that I converted to a script so I could send it Tekniker so they could run it from the command line. I really have nowhere else to put it. I'll add a description though...